src/_variables.tsx

```diff
@@ -0,0 +1,7 @@
+export const variables = {
```

Comment:
we dont need underscore for js files :) 
https://stackoverflow.com/questions/34889962/why-put-in-front-of-the-file-name-or-in-scss-css